app/src/main/java/com/example/taskwise_eventmaster/presentation/home_page/MenuPartOfHomeScreen.kt

```diff
@@ -58,11 +68,20 @@ fun MenuPart(
             }
         }
 
-        imageCards[0].onClickImage = goToTaskPage
-        imageCards[1].onClickImage = goToGoalsPage
-        imageCards[2].onClickImage = goToEventsPage
-        imageCards[3].onClickImage = goToPlanningViewPage
+        val screens = listOf(TASK, GOAL, EVENTS, PLANNING_VIEW)
```

Comment:
I'd argue that there is no need to store those image cards in classes and iterate over them to present them, because it does not look like those screens could dynamically change or even change at all.

Reply:
If those screens are set in stone and will always be the 4 screens we know of today, we can just put them as composables and not abstract them away, just so it's clearly visible what is going on in this screen by looking at the code.